src/html/fetch-markdown.js

```diff
@@ -10,11 +10,12 @@
  * governing permissions and limitations under the License.
  */
 const { inspect } = require('util');
-const client = require('request-promise-native');
 const URI = require('uri-js');
 const { setdefault } = require('ferrum');
+const { fetch, TimeoutError } = require('@adobe/helix-fetch');
```

Comment:
can we change the client in a separate PR? too many moving targets....

Reply:
Your wish is my command: https://github.com/adobe/helix-pipeline/pull/582

I'll update this PR once the other one is merged in master